meetings/agenda.md

```diff
@@ -7,3 +7,48 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 
 ## Items to discuss
 
+### Components to be potentially moved to security-only status
+
+Proposal: these components should (probably) be clearly labeled so that maintainers know that only security
+issues are to be fixed, if they occur.
+
+ * laminas-db
+ * laminas-developer-tools
+ * laminas/laminas-oauth
+ * laminas/laminas-xml
+ * laminas/laminas-composer-autoloading
+ * laminas/laminas-auradi-config
+ * laminas/laminas-development-mode
+ * laminas/documentation
+ * laminas/laminas-barcode
+ * laminas/laminas-config-aggregator-parameters
+ * laminas/laminas-config
+ * laminas/laminas-config-aggregator-modulemanager
+ * laminas/laminas-console
+ * laminas/laminas-dom
+ * laminas/laminas-crypt
+ * laminas/laminas-mvc-console
+ * laminas/laminas-json
+ * laminas/laminas-http
+ * laminas/laminas-memory
+ * laminas/laminas-loader
+ * laminas/laminas-servicemanager-di
+ * laminas/laminas-json-server
+ * laminas/laminas-mime
+ * laminas/laminas-log
+ * laminas/laminas-math
+ * laminas/laminas-xml2json
+ * laminas/laminas-soap
```

Comment:
We have a lot of folks who depend on this, as there are not many good SOAP solutions for PHP out there. With the updates to laminas-server, it may make sense to revisit this one and update it.